test/xpu_api/random/conformance_tests/common_for_conformance_tests.hpp

```diff
@@ -62,8 +64,36 @@ typename Engine::scalar_type test(sycl::queue& queue) {
             return 0;
         }
     }
-
+    std::cout << "\n\t\tres sycl: " << dpstd_samples[REF_SAMPLE_ID] << std::endl;
     return dpstd_samples[REF_SAMPLE_ID];
+
+#else
```

Comment:
please, remove #if1 later

Reply:
I don't need that anymore, I think. Removed.